txclib/cmdline.py

```diff
@@ -113,9 +113,8 @@ def main(argv=None):
         logger.error("SSl error %s" % e)
         sys.exit(1)
     except utils.UnknownCommandError:
-        logger.error("tx: Command %s not found" % cmd)
-    except SystemExit:
-        sys.exit()
+        logger.error("Command %s not found" % cmd)
+        sys.exit(1)
```

Comment:
@fathineos there's a [bare except clause](https://docs.python.org/2/howto/doanddont.html#except) just below these lines. Previously the code was catching `SystemExit`, but now as far as I can tell it will be caught by the code below which will log an error. This will happen even when the exit is not unsuccessful. So, either you keep the `SystemExit` clause as before or you fix the bare except below (the latter is preferable).